websites/P/ProBot/presence.ts

```diff
@@ -9,34 +9,165 @@ presence.on("UpdateData", () => {
 		startTimestamp: browsingTimestamp
 	};
 
-	if (document.location.pathname.includes("/dashboard")) {
-		presenceData.details = "Managing Server";
-		presenceData.state = (
-			document.querySelector("div.user-profile > div > div") as HTMLElement
-		).textContent;
+	if (document.location.pathname === "/")
+		presenceData.details = "Visiting the Main Page";
+	else if (document.location.pathname.includes("/dashboard")) {
+		presenceData.details = "Visiting the Dashboard";
+		presenceData.buttons = [
+			{
+				label: "Go to Dashboard",
+				url: document.URL
+			}
+		];
 	} else if (document.location.pathname.includes("/commands")) {
 		presenceData.details = "Viewing a page:";
 		presenceData.state = "Commands";
-	} else if (document.location.pathname.includes("/premium"))
-		presenceData.details = "Viewing Premium";
-	else if (document.location.pathname.includes("/bgs")) {
-		presenceData.details = "Managing the";
-		presenceData.state = "profile backgrounds";
-	} else if (document.location.pathname.includes("/id")) {
-		presenceData.details = "Managing the";
-		presenceData.state = "ID backgrounds";
+		presenceData.buttons = [
+			{
+				label: "View ProBot Commands",
+				url: document.URL
+			}
+		];
+	} else if (document.location.pathname.includes("premium")) {
+		if (document.location.pathname.includes("/manage"))
+			presenceData.details = "Managing their Premium";
+		else presenceData.details = "Reading more about Premium";
+	} else if (document.location.pathname.includes("/store")) {
+		presenceData.details = "Managing their";
+		if (document.location.pathname.includes("/profile"))
+			presenceData.state = "Profile Backgrounds";
+		else if (document.location.pathname.includes("/rank"))
+			presenceData.state = "ID Backgrounds";
+		else presenceData.state = "purchased items";
 	} else if (document.location.pathname.includes("/badges")) {
-		presenceData.details = "Managing the";
-		presenceData.state = "profile badges";
+		presenceData.details = "Managing their";
+		presenceData.state = "Profile Badges";
 	} else if (document.location.pathname.includes("/xp")) {
-		presenceData.details = "Viewing a page:";
-		presenceData.state = "Top 100 by XP";
+		presenceData.details = "Viewing top 100 people";
+		presenceData.state = "with the most XP";
+		presenceData.buttons = [
+			{
+				label: "Visit the page",
+				url: document.URL
+			}
+		];
 	} else if (document.location.pathname.includes("/credits")) {
-		presenceData.details = "Viewing a page:";
-		presenceData.state = "Richest 100 Billionaires";
+		presenceData.details = "Viewing top 100 people";
+		presenceData.state = "with the most Credits";
+		presenceData.buttons = [
+			{
+				label: "Visit the page",
+				url: document.URL
+			}
+		];
+	} else if (document.location.pathname.includes("/daily")) {
+		presenceData.details = "Claiming their daily Credits";
+		presenceData.buttons = [
+			{
+				label: "Claim your daily reward!",
+				url: document.URL
+			}
+		];
```

Comment:
you shouldn't assume everyone viewing this wants to claim their daily reward. I think this button is useless